components/Footer.js

```diff
@@ -0,0 +1,31 @@
+import HouseLine from "@/public/Icons/home-4-line.svg";
+import styled from "styled-components";
+import PlantFavIcon from "@/public/Icons/plant-line.svg";
+import Link from "next/link";
+
+export default function Footer() {
+  return (
+    <StyledFooter>
+      <StyledLink href={"/"} aria-label="Go to homepage">
```

Comment:
_Just a little note:_
Curly braces `{}` are unnecessary for plain strings
```suggestion
      <StyledLink href="/" aria-label="Go to homepage">
```